torch/autograd/function.py

```diff
@@ -533,12 +534,29 @@ def vmap(info, in_dims, *args):
 
     @classmethod
     def apply(cls, *args, **kwargs):
+        def bind_default_args(func, is_def_ctx, *args, **kwargs):
+            arguments: List[Any]
+
+            arguments = list(args)
```

Comment:
nit: why have the typing annotation on a separate line? (Also, with the simplification suggested below, we wont' even need a separate arguments list variable.)

Reply:
Done